networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/realization.py

```diff
@@ -34,19 +36,23 @@ def __init__(
 
         self.mngr_provider = mngr_provider
         self.plcy_provider = plcy_provider
+        self.migration_tracker = provider.MigrationTracker(self.plcy_provider)
+
 
-        self.PAUSE_REALIZATION = False
         self.AGE = int(time.time())
 
         LOG.info("Detected NSX-T %s version.", self.mngr_provider.client.version)
 
+        self.mp_to_policy_completed = any([t for t in self.plcy_provider.zone_tags if t.get(
+            "scope") == NSXV3_MP_MIGRATION_SCOPE and t.get("tag") == NSXV3_MIGRATION_SUCCESS_TAG])
+
         # Enable MP-to-Policy migration if force_mp_to_policy=True
         # It is used as a flag for using Policy API completely or not
         # in case migration canceled or failed this flag will be False
         # TODO: After completing the transition to NSX Policy API (ONLY if successful!), deprecate this flag
-        self.USE_POLICY_API = self._check_mp2policy_support()
+        self.USE_POLICY_API = self.mp_to_policy_completed or self._check_mp2policy_support()
 
-        if self.USE_POLICY_API:
+        if self.USE_POLICY_API and not self.mp_to_policy_completed:
```

Comment:
what about    if  self.mp_to_policy_completed: return to make it clear that migration has already happend.